google/cloud/storage/testing/mock_client.h

```diff
@@ -185,14 +185,26 @@ class MockStreambuf : public internal::ObjectWriteStreambuf {
   MOCK_METHOD(std::uint64_t, next_expected_byte, (), (const, override));
 };
 
-/// Create a client configured to use the given mock.
+/**
+ * Create a client configured to use the given mock.
+ *
+ * Unless you specifically need to mock the behavior of retries, prefer
+ * #google::cloud::storage::testing::UndecoratedClientFromMock
```

Comment:
Typically if you say `UndecoratedClientFromMock()` (note the parenthesis) Doxygen will automatically link the function.

Reply:
Fixed, thanks.